blueprint/api.apib

```diff
@@ -2174,6 +2234,55 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + data(Slugged Route Organization Resource)
 + include JSON API Version
 
+## Stripe Card Attributes (object)
++ brand: (enum[string]) - Card brand.
+    + Members
+        + `Visa`
+        + `American Express`
+        + `MasterCard`
+        + `Discover`
+        + `JCB`
+        + `Diners Club`
+        + `Unknown`
++ cvc_check: (enum[string]) - If a CVC was provided, results of the check.
+    + Members
+        + `pass`
+        + `fail`
+        + `unavailable`
+        + `unchecked`
++ exp_month: `12` (number)
++ exp_year: `2017` (number)
```

Comment:
I also don't think this value needs backticks around it if it's just an integer. Not sure, but we should check some of the other examples.